Sources/Apollo/RecordSet.swift

```diff
@@ -14,6 +14,14 @@ public struct RecordSet {
     storage.removeValue(forKey: key)
   }
 
+  public mutating func removeRecords(matching pattern: CacheKey) {
+    for (key, _) in storage {
+      if key.range(of: pattern, options: .caseInsensitive) != nil {
```

Comment:
This comparison
* doesn't use `contains(:)` because that's case sensitive
* doesn't use `localizedCaseInsensitiveContains(:) because the locale of the device is not guaranteed to be the same as any app-specific locale

Reply:
Right now this is logically equivalent to `key.uppercased().contains(pattern.uppercased())`. I'm undecided whether we should be using the option `.diacriticInsensitive` too?

Reply:
my recollection is you can't use diacritics in graphql keys, but that's worth double-checking

Reply:
GraphQL spec only allows ASCII characters.

Reply:
OK - then I'm going to leave it as is with the `range(of:options:)` implementation. I think it's more explicit in what it's doing vs. `contains(:)` with both `uppercased()`.